examples/oed_utils.jl

```diff
@@ -185,6 +183,20 @@ function add_to_min(x, u)
     return x
 end
 
+function add_to_min2(x,u)
+    perm = sortperm(x)
+    
+    for i in perm
+        if x[i] < u[i]
+            x[i] += 1
+            break
+        else
+            continue
+        end
```

Comment:
when there is a loop with a break statement, it interrupts the loop so the rest doesn't need an else clause with a continue, one can write:
```
for i in perm
    if condition
        break
    end # no else
    # continue loop naturally
end